app/Providers/RouteServiceProvider.php

```diff
@@ -35,6 +38,18 @@ public function boot(): void
             Route::middleware('web')
                 ->group(base_path('routes/web.php'));
         });
+        Route::bind('pendingSubmission', function (string $value) {
+            $submission = Submission::whereNull('doctor_id')->find($value);
+
+            if (! $submission) {
+                throw new HttpException(
```

Comment:
```suggestion
                if (! $submission) {
                    abort(Response::HTTP_FORBIDDEN, 'The submission is already assigned to a doctor');
                }
```

Laravel ya esta preparado para que si llamas al abort crea el response adecuado.

Reply:
Nunca usemos helpers a menos que no haya otra opcion, como regla general!
Ni abort, ni request, ni nada de eso. Lo magico siempre de ultimo 

Yo se que @gianfranco-rocco ama lo magico muaja

Reply:
😶‍🌫️

Reply:
Ya de por si el Custom Binding es muy magico para mi gusto. En este caso no molesta porque el mensaje es claro. Pero a nivel codigo cuesta entender de donde salio el error.